ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ReviewConfig.tsx

```diff
@@ -40,7 +39,7 @@ function ReviewConfig({ clusters, profiles }: ProfileSelectionProps) {
             .filter((item): item is T => item !== undefined);
     }
 
-    const selectedClusters = findById<ClusterScopeObject>(formikValues.clusters, clusters);
+    const selectedClusters = findById<ComplianceIntegration>(formikValues.clusters, clusters);
```

Comment:
Brad, does change from `id` to `clusterId` in `ComplianceIntegration` affect `findById` helper function?